projects/import-azuredevops-client/package.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "import-azuredevops-client",
   "version": "0.0.1",
+  "dependencies": {
+    "tslib": "^2.0.0"
+  },
   "peerDependencies": {
     "@angular/common": "^9.0.3",
-    "@angular/core": "^9.0.3",
-    "tslib": "^1.10.0"
+    "@angular/core": "^9.0.3"
```

Comment:
This should probably be merged with a similar update for the import-xxx-clients.

Reply:
The providers are generated using swagger-codegen and are currently generated targetting Angular 9.0.3 (see the `generate-rest-client.ps1`

I've intentionally left out these changes from this PR as that can be changed in a later step.

Reply:
Created PR: #30